db/migrate/20170717140915_add_numeric_columns_to_container_quota_items.rb

```diff
@@ -0,0 +1,44 @@
+class AddNumericColumnsToContainerQuotaItems < ActiveRecord::Migration[5.0]
+  class ContainerQuotaItem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def up
+    add_column :container_quota_items, :quota_desired_float,  :float
+    add_column :container_quota_items, :quota_desired_int,    :bigint
```

Comment:
@Fryguy We are trying to persist the `quantity` type from kubernetes which is a string that represents some type of number https://github.com/kubernetes/apimachinery/blob/master/pkg/api/resource/quantity.go#L41.
We are trying to ease the overhead on reporting when aggregating or averaging these values by saving them directly as bigint\float. This way instead of parsing strings on the fly when generating reports we already have their numeric value. Since we dont know if the value will be a float or an int we add both columns and fill the appropriate one when parsing the string.
Do you think we can find a better trick here?

cc @simon3z